packages/functions/src/common/BookAppointmentHelper.ts

```diff
@@ -11,43 +11,35 @@ import {
   FunctionsApi,
   Hospital,
   MANUAL_DONOR_ID,
+  MinimalDonorDetailsForAppointment,
 } from "@zm-blood-components/common";
 import { dbAppointmentToBookedAppointmentApiEntry } from "../utils/ApiEntriesConversionUtils";
 import { notifyOnAppointmentBooked } from "../notifications/BookAppointmentNotifier";
-import { BookAppointmentStatus } from "@zm-blood-components/common/src/functions-api";
 import { DbDonor } from "../function-types";
 
 const WEEKS_BUFFER = 0;
 
-export async function BookAppointment(
-  request: FunctionsApi.BookAppointmentRequest,
+export async function bookAppointment(
+  donorId: string,
   isDonor: boolean,
```

Comment:
Is this still needed?

Reply:
what do you mean?

Reply:
`isDonor == true` when `coordinatorId == undefined` - So you don't need both fields

Reply:
oh write